keras_nlp/layers/masked_lm_mask_generator.py

```diff
@@ -147,11 +147,7 @@ def call(self, inputs):
             # convert dense to ragged.
             inputs = tf.RaggedTensor.from_tensor(inputs)
 
-        (
```

Comment:
unnecessary code formatting change.

Once you have installed black, to format the code
Go to root folder -> run `./shell/format.sh` and `./shell/lint.sh` to format the files correctly.